From: =?utf-8?q?IOhannes_m_zm=C3=B6lnig?= <umlaeute@debian.org>
Date: Sat, 30 May 2020 10:16:59 +0200
Subject: Always rely on the version from debian changelog

Forwarded: Not-Needed
Last-Update: 2020-05-27

We generate libs/ardour/revision.cc in debian/rules.
Adapt the wscript to parse our custom format. To avoid
confusion with any existing .git directory on a packager's
machine, disable the git version check and pretend it's a
tarball.
---
 wscript | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- ardour.orig/wscript
+++ ardour/wscript
@@ -237,7 +237,7 @@
         return 'sequoia'
 
 
-def set_version (from_file = False):
+def set_version (from_file = True):
     def sanitize(s):
         # round-trip to remove anything in the string that is not encodable in
         # ASCII, yet still keep a real (utf8-encoded internally) string.
@@ -278,7 +278,7 @@
 
     parts = sanitize(rev).split ('.', 1)
     MAJOR = parts[0]
-    other = parts[1].split('-', 1)
+    other = parts[1].split('.', 1)
     MINOR = other[0]
     if len(other) > 1:
         MICRO = other[1].rsplit('-',1)[0].replace('-','.')
@@ -298,7 +298,7 @@
 def create_stored_revision():
     set_version ()
     rev = ""
-    if os.path.exists('.git'):
+    if False or os.path.exists('.git'):
         rev, rev_date = fetch_git_revision_date()
         print("Git version: " + rev + "\n")
     elif os.path.exists('libs/ardour/revision.cc'):
